packages/page/src/Application/Message/MovePageMessage.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Page\Application\Message;
+
+class MovePageMessage
+{
+    /**
+     * @param array{
+     *     uuid?: string
+     * } $identifier
+     * @param array{
+     *     uuid?: string
+     * } $destinationIdentifier
+     */
+    public function __construct(
+        private array $identifier,
+        private array $destinationIdentifier,
```

Comment:
Not sure about destination here, kind of missing `parent` in this variable names.

```suggestion
        private array $parentIdentifier,
```